_posts/2017-07-24-code-gov-case-study.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How the Code.gov team uses the U.S. Web Design Standards
+tags:
+- web design standards
+- design
+- case study
+- design
+---
+The U.S. Web Design Standards team continues to sit down with various agencies who are using the Standards. In this sixth post in our series, we talked to Olivier Kamanda, project manager for [code.gov](https://code.gov) to learn how that team adopted the Standards. [Code.gov](https://code.gov) is a new platform designed to be the home for federal source code--a platform for improving the quality of government software. It launched in November 2016.
+
+**Standards team:** How did you hear about the U.S. Web Design Standards?
+
+**Olivier Kamanda:** I inherited the design work from another Presidential Innovation Fellow (PIF) who designed [vote.gov](https://vote.gov) and [apps.gov](https://apps.gov), which all use the Standards to some degree.
+
+**Standards team:** Why did the team decide to use the Standards?
+
+**Olivier Kamanda:** When I joined the project, the Standards were already adopted. The original designer liked that the Standards gave the sites a modern, clean look that didn’t look like other government sites.
+
+**Standards team:** How did you integrate the Standards into your work?
+
+**Olivier Kamanda:** The site is built is nodejs and angular2, which allowed us to use many of the components. We’ve used the tables, headings and menus. For our page layout, we use Bootstrap in conjunction with the Standards.
```

Comment:
Minor nit--it looks like Node.js's website spells it "Node.js", and angular2 should be "Angular 2"?  Up to you I guess, though it was really seeing "Bootstrap" capitalized that made "nodejs" and "angular2" seem out of place.